action.yml

```diff
@@ -96,4 +99,4 @@ branding:
   icon: 'icon.svg'
 runs:
   using: 'node12'
-  main: 'lib/main.js'
+  main: 'lib/main.js'
```

Comment:
Please make sure a new line is present at the end of each file.

Reply:
Sure. Sorry for the mistakes.